github/libp2p.yml

```diff
@@ -3986,6 +3986,9 @@ repositories:
             - Compile with select features (mdns tcp dns async-std)
             - Compile with select features (mdns tcp dns tokio)
             - IPFS Integration tests
+            - run-ping-interop-cross-implementation / Run a test with different
+              versions
```

Comment:
Oh I am not sure about this newline. I think GitHub might not recognize that ...

Reply:
Have a look at the checks that terraform will add - https://github.com/libp2p/github-mgmt/pull/79#issuecomment-1323273764. Looks like the string is correctly interpreted when we perform yaml parsing.